src/test/java/org/takes/tk/TkProxyTest.java

```diff
@@ -37,19 +39,125 @@
  * @author Dragan Bozanovic (bozanovicdr@gmail.com)
  * @version $Id$
  * @since 0.25
- * @todo #377:30min/DEV We need more tests for TkProxy.
+ *
  *  The tests should verify the different HTTP methods (GET, POST, etc),
```

Comment:
@aschworer usually, we write class purpose at the very beginning, not in the end